docs/usage/webhook/deploy.rst

```diff
@@ -0,0 +1,152 @@
+.. _usage_ffmpeg_deploy:
+
+Deploying with the HTTP Service
+===============================
+
+In this tutorial we will deploy a dataflow(ffmpeg dataflow) which converts a video to gif over an HTTP service. We'll
+also see how to deploy the same in a docker container. Finally in :ref:`usage_ffmpeg_deploy_serve`
+we'll setup another HTTP service which waits on GitHub webhooks to rebuilt and deploy the ffmpeg dataflow.
+
+.. note::
+
+    All the code for this example is located under the
+    `examples/ffmpeg <https://github.com/intel/dffml/blob/master/examples/ffmpeg/>`_
+    directory of the DFFML source code.Rest of the tutorial is ran from the root of
+    this folder.
+
```

Comment:
Let's have them install `dffml-service-http` and `dffml-config-yaml` after the `.. note`